src/main/g8/src/main/scala/$package__packaged$/ServerStream.scala

```diff
@@ -0,0 +1,18 @@
+package $package$
+
+import cats.effect.{ConcurrentEffect, Effect, ExitCode, IO, IOApp, Timer, ContextShift}
+import cats.implicits._
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.http4s.implicits._
+import fs2.Stream
+
+object ServerStream {
+  def helloWorldApp[F[_]: Effect] = HelloWorldRoutes.routes[F].orNotFound
+
+  def stream[F[_]: ConcurrentEffect](implicit T: Timer[F], C: ContextShift[F]): Stream[F, Nothing] =
+    BlazeServerBuilder[F]
+      .bindHttp(8080, "0.0.0.0")
+      .withHttpApp(helloWorldApp)
+      .serve
+      .drain
```

Comment:
I've lost track.  Why are we `.drain`ing here, and not entirely covered by the `IOApp`?

Reply:
Was @systemfw suggestion in chat.

Reply:
actually no, I didn't mean this.
I meant that using `IOApp` directly and letting the user `.compile.drain` (not `drain`) is better, because `.as(ExitCode)` on `IO` makes more sense than on Stream

Reply:
Gotcha, will fix.